profiles/serializer/normalizer.py

```diff
@@ -0,0 +1,60 @@
+from functools import singledispatch
+from typing import Any
+
+from profiles.models import Affiliation, EmailAddress, Name, Profile
+
+
+@singledispatch
+def normalize(value: Any) -> str:
+    return str(value)
+
+
+@singledispatch
+def normalize_snippet(value: Any) -> str:
+    return str(value)
+
+
+@normalize.register(Profile)
+def normalize_profile(profile: Profile) -> dict:
+    data = normalize_profile_snippet(profile)
+    data['emailAddresses'] = [normalize_snippet(email) for email in profile.email_addresses]
+    data['affiliations'] = [normalize_snippet(aff) for aff in profile.get_affiliations()]
```

Comment:
We'll need to ignore email addresses and affiliations that are `restricted`. (We might want them in the future, but only for API users with permission.)

Reply:
I have now added default functionality to filter out restricted email addresses and affiliations unless you flag to included them, which could be tied into another endpoint for users with said permission in the future.